### PR TITLE
add a few CRAN packages needed in bioconda

### PR DESCRIPTION
**Pull request by dpryan79**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dpryan79** [comment]
The wrapper was used to generate them. They're all generic, so only Linux should pass.

**Comment by dpryan79** [comment]
@conda-froge/help-r These are ready for review. They're all generic, so I expect failures on windows are still expected.

File: recipes/r-reldist/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    - "\"%R%\" -e \"library('reldist')\""  # [win]
+
+about:
+  home: http://www.stat.ucla.edu/~handcock/RelDist
+  license: GPL-3
+  summary: Tools for the comparison of distributions. This includes nonparametric estimation
+    of the relative distribution PDF and CDF and numerical summaries as described in
+    "Relative Distribution Methods in the Social Sciences" by Mark S. Handcock and Martina
+    Morris, Springer-Verlag, 1999, Springer-Verlag, ISBN 0387987789.
+  license_family: GPL3
```
Comment (dbast) [inline recipes/r-reldist/meta.yaml]: why is the generator not adding `license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'`

File: recipes/r-rmagic/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+about:
+  home: https://CRAN.R-project.org/package=Rmagic
+  license: GPL-2
+  summary: 'MAGIC (Markov affinity-based graph imputation of cells) is a method for addressing
+    technical noise in single-cell data, including under-sampling of mRNA molecules,
+    often termed "dropout" which can severely obscure important gene-gene relationships.
+    MAGIC shares information across similar cells, via data diffusion, to denoise the
+    cell count matrix and fill in missing transcripts. Read more: van Dijk et al. (2018)
+    <DOI:10.1016/j.cell.2018.05.061>.'
+  license_family: GPL2
```
Comment (dbast) [inline recipes/r-rmagic/meta.yaml]: and why is the generator not adding `license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'`

File: recipes/r-saver/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    - "\"%R%\" -e \"library('SAVER')\""  # [win]
+
+about:
+  home: https://github.com/mohuangx/SAVER
+  license: GPL-2
+  summary: An implementation of a regularized regression prediction and  empirical Bayes method
+    to recover the true gene expression profile in  noisy and sparse single-cell RNA-seq
+    data. See Huang M, et al (2018)  <doi:10.1038/s41592-018-0033-z> for more details.
+  license_family: GPL2
+  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
```
Comment (dbast) [inline recipes/r-saver/meta.yaml]: Fine here.

File: recipes/r-sybil/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+about:
+  home: http://www.cs.hhu.de/lehrstuehle-und-arbeitsgruppen/computational-cell-biology/software/sybil.html
+  license: GPL-3
+  summary: This Systems Biology Package (Gelius-Dietrich et. al. (2012) <doi:10.1186/1752-0509-7-125>)
+    implements algorithms for constraint based analyses of metabolic networks, e.g.
+    flux-balance analysis (FBA), minimization of metabolic adjustment (MOMA), regulatory
+    on/off minimization (ROOM), robustness analysis and flux variability analysis. The
+    package is easily extendable for additional algorithms. Most of the current LP/MILP
+    solvers are supported via additional packages.
+  license_family: GPL3
```
Comment (dbast) [inline recipes/r-sybil/meta.yaml]: again `license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'`

**Comment by dbast** [comment]
some `license_file`s and this should be fine.

File: recipes/r-reldist/meta.yaml (hunk 1 above)
Comment (dpryan79) [reply recipes/r-reldist/meta.yaml]: It probably doesn't exist. Last I looked CRAN didn't actually enforce this (maybe it does for new packages).

Comment (dpryan79) [reply recipes/r-reldist/meta.yaml]: Hmm, in this case it's just `LICENSE`. I'll try to manually add these in.

**Comment by dpryan79** [comment]
I've added the licenses in, we'll see if it passes then (it's been hit or miss in the past).

File: recipes/r-reldist/meta.yaml (hunk 1 above)
Comment (dbast) [reply recipes/r-reldist/meta.yaml]: ah. `'{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'` is referring to a file provided r-base package, not by the cran tarball of reldist.

Comment (dbast) [reply recipes/r-reldist/meta.yaml]: The skeleton creator normally add those lines by matching the cran meta data... see https://github.com/conda/conda-build/blob/master/conda_build/skeletons/cran.py#L1430 till line 1441 ... sometimes this seems not to work.

**Comment by dbast** [comment]
The hit or miss problem should only exist when e.g. adding `license_file: LICENSE` ... Adding `license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/...'` should always work, see comment above.

**Comment by dpryan79** [comment]
Yeah, it must have been the LICENSE file that I was thinking of then.

**Pull request merged**